src/lib/config.js

```diff
@@ -0,0 +1,19 @@
+require("dotenv").config();
+
+module.exports = {
+  API: {
+    BASE_URL: process.env.API_BASE_URL || "http://localhost:5007",
+    PATHS: {
+      AUTHORIZE: "/authorize",
+    },
+  },
+  APP: {
+    BASE_URL: process.env.EXTERNAL_WEBSITE_HOST || "http://localhost:5000",
```

Comment:
This is the website as it is being accessed by the user, so the port should also be set as `5010` c/o `config` & `Dockerfile`